okcli/completion_cache.py

```diff
@@ -0,0 +1,18 @@
+import hashlib
```

Comment:
Can we have some test-coverage for these, please? Nothing too involved - just a few simple unit tests.